freqtrade/rpc/telegram.py

```diff
@@ -262,19 +291,52 @@ def _format_sell_msg(self, msg: Dict[str, Any]) -> str:
                                    ' / {profit_fiat:.3f} {fiat_currency})').format(**msg)
         else:
             msg['profit_extra'] = ''
-
+        msg['currency'] = msg['pair'].split('/')[0]
         message = ("{emoji} *{exchange}:* Selling {pair} (#{trade_id})\n"
-                   "*Profit:* `{profit_ratio:.2%}{profit_extra}`\n"
+                   "*Unrealized Profit:* `{profit_ratio:.2%}{profit_extra}`\n"
                    "*Buy Tag:* `{buy_tag}`\n"
                    "*Sell Reason:* `{sell_reason}`\n"
                    "*Duration:* `{duration} ({duration_min:.1f} min)`\n"
-                   "*Amount:* `{amount:.8f}`\n"
+                   "*Amount:* `{amount:.8f}` {currency}\n"
                    "*Open Rate:* `{open_rate:.8f}`\n"
                    "*Current Rate:* `{current_rate:.8f}`\n"
                    "*Close Rate:* `{limit:.8f}`").format(**msg)
 
         return message
 
+    def _format_sell_fill_msg(self, msg: Dict[str, Any]) -> str:
+        import pprint
+        pprint.pprint(msg)
```

Comment:
debug msg?

Reply:
Whops yes will remove this, was for debugging